src/Reference/Commitments/SFA.DAS.Payments.Reference.Commitments.IntegrationTests/GivenApiAvailable/WhenReadingEvents.cs

```diff
@@ -637,6 +637,100 @@ public void ThenItShouldAddEventPriceHistoryWhereEffectiveFromIsBeforeEffectiveT
             Assert.AreEqual(1, history.Length);
         }
 
+        [Test]
+        public void ThenItShouldNotAddEventWhereEffectiveFromIsBeforeEffectiveTo()
```

Comment:
looks like the test names are the wrong way around?

Reply:
they certainly seem to be! Code blind when I did that the day before my holiday. I've swapped them around and properly named them